AirCasting/Map/GoogleMapView.swift

```diff
@@ -87,15 +87,22 @@ struct GoogleMapView: UIViewRepresentable {
 
     func colorPolyline(point: PathPoint) -> UIColor {
         let measurement = point.measurement
+        guard thresholds.count >= 5 else { return UIColor.white }
+        
+        let veryLow = Float(thresholds[0])
+        let low = Float(thresholds[1])
+        let medium = Float(thresholds[2])
+        let high = Float(thresholds[3])
+        let veryHigh = Float(thresholds[4])
```

Comment:
Seams like array is not a good model for the `thresholds` if it requires 4 elements always. Maybe can be changed to a light understandable struct?